package.json

```diff
@@ -72,7 +72,7 @@
     "rimraf": "^2.5.4",
     "tslint": "latest",
     "tslint-test-config-non-relative": "file:test/external/tslint-test-config-non-relative",
-    "typescript": "^2.3.0"
+    "typescript": "next"
```

Comment:
not sure if it makes sense to use typescript@next in development while having a yarn.lock. I guess there'll be many merge conflicts just because of the typescript version in yarn.lock